src/System.Numerics.Vectors/src/System/Numerics/Register.tt

```diff
@@ -5,7 +5,7 @@
 <#@ import namespace="System" #>
 <#@ import namespace="System.Linq" #>
 <#@ import namespace="System.Runtime.InteropServices" #>
-<#@ import namespace="System.Diagnostics.Contracts" #>
```

Comment:
I believe that the Assert in this file is something checked at template instantiation time not runtime.  Contract.cs doesn't have any Debug or Contract asserts in it and then test refers to variables introduced by GenerationConfiguration.ttinclude.